python/GettingStarted.ipynb

```diff
@@ -24,45 +24,42 @@
+      "Requirement already satisfied: whylabs-datasketches>=2.0.0b7 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from whylogs==0.1.10->-r ./requirements.txt (line 6)) (2.0.0b7)\r\n",
+      "Requirement already satisfied: pandas>1.0 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from whylogs==0.1.10->-r ./requirements.txt (line 6)) (1.2.1)\r\n",
+      "Requirement already satisfied: marshmallow>=3.7.1 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from whylogs==0.1.10->-r ./requirements.txt (line 6)) (3.10.0)\r\n",
+      "Requirement already satisfied: botocore>=1.17.44 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from whylogs==0.1.10->-r ./requirements.txt (line 6)) (1.20.1)\r\n",
+      "Requirement already satisfied: fsspec>=0.6.0 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from s3fs==0.4.2->whylogs==0.1.10->-r ./requirements.txt (line 6)) (0.8.5)\r\n",
+      "Requirement already satisfied: s3transfer<0.4.0,>=0.3.0 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from boto3->-r ./requirements.txt (line 1)) (0.3.4)\r\n",
+      "Requirement already satisfied: jmespath<1.0.0,>=0.7.1 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from boto3->-r ./requirements.txt (line 1)) (0.10.0)\r\n",
+      "Requirement already satisfied: urllib3<1.27,>=1.25.4 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from botocore>=1.17.44->whylogs==0.1.10->-r ./requirements.txt (line 6)) (1.26.3)\r\n",
+      "Requirement already satisfied: pytz>=2017.3 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from pandas>1.0->whylogs==0.1.10->-r ./requirements.txt (line 6)) (2021.1)\r\n",
+      "Requirement already satisfied: six>=1.9 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from protobuf>=3.12.2->whylogs==0.1.10->-r ./requirements.txt (line 6)) (1.15.0)\r\n",
+      "Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.3 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from matplotlib->-r ./requirements.txt (line 4)) (2.4.7)\r\n",
+      "Requirement already satisfied: cycler>=0.10 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from matplotlib->-r ./requirements.txt (line 4)) (0.10.0)\r\n",
+      "Requirement already satisfied: kiwisolver>=1.0.1 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from matplotlib->-r ./requirements.txt (line 4)) (1.3.1)\r\n",
+      "Requirement already satisfied: pillow>=6.2.0 in /Users/alex/miniconda3/envs/whylogs-example-v3/lib/python3.8/site-packages (from matplotlib->-r ./requirements.txt (line 4)) (8.1.0)\r\n"
      ]
     }
    ],
    "source": [
-    "import sys\n",
```

Comment:
This is required. The notebook is broken without this.

Reply:
fixed, thank you!